aspnetcore/mvc/razor-pages/razor-pages-authorization-conventions/sample/Pages/Index.cshtml

```diff
@@ -0,0 +1,121 @@
+﻿@page
+@model IndexModel
+@inject Microsoft.AspNetCore.Http.IHttpContextAccessor HttpContextAccessor;
+@{
+    ViewData["Title"] = "Home page";
+}
+
+@if (HttpContextAccessor.HttpContext.User.Identity.IsAuthenticated)
+{
+    <div class="row">
+        <div class="col-md-12">
+            <div style="font-size:2.4em;margin-top:6px">Hello @HttpContextAccessor.HttpContext.User.Claims.FirstOrDefault(c => c.Type == "FullName")?.Value!</div>
```

Comment:
Consider replacing the inline style with a custom CSS class you create.

Reply:
Since it was just a one-off, I inlined it. I frequently inline for one-offs (single page, single style), especially on landing pages (u know ... going for broke on speed). However, I do have the *site.css*/*site.min.css*, so I'll move it over there and call it the `welcome` class or something.